explainer.md

```diff
@@ -144,7 +143,6 @@ If the media item url will not resume playback at the correct time, then the `ta
     "url": "https://example.org/testuser"
   },
   "datePublished": "2020-01-27",
-  "description": "This is a video that is about a bunny."
```

Comment:
Why remove these descriptions?

Reply:
They are not needed since they are not part of the spec